dcos-deploy-script/Makefile

```diff
@@ -0,0 +1,127 @@
+DCOS_ENDPOINT ?= <DC/OS URL>
+APIGATEWAY_DNS ?= <gateway endpoint>
+OPENWHISK_UNIVERSE ?= <Universe URL>
+WSK_NAMESPACE ?= guest
+WSK_AUTH_TOKEN ?= <enter token here>
+
+cli:
```

Comment:
like with the universe docker bellow, WDYT about exposing a docker image with the dcos CLI and potentially wsk CLI that could be used to execute these scripts in any environments ?

Reply:
yep +1